plot-doc/plot/scribblings/utils.scrbl

```diff
@@ -582,3 +582,136 @@ Convert @racket[plot-time]s to real seconds, and vice-versa.
                  (plot-time+ (plot-time 32 0 12 1)
                              (plot-time 32 0 14 1))]
 }
+
+@section{Plot Metrics}
+
+@definterface[plot-metrics<%> ()]{
+
+  The @racket[plot-metrics<%>] interface allows obtaining plot area positions
+  on the plots returned by @racket[plot], @racket[plot-snip],
+  @racket[plot-bitmap], @racket[plot/dc], as well as their 3D variants,
+  @racket[plot3d], @racket[plot3d-snip], @racket[plot3d-bitmap] and
+  @racket[plot3d/dc].  All plot objects returned by these functions implement
+  this interface.
+
+  In addition to this, the plots created by @racket[plot-snip] and
```

Comment:
What is this 'own' functionality for plot(3d)-snip?
Or did you mean plot(3d)-pict?

Reply:
Yes, I meant `plot-pict` and `plot3d-pict`.  I fixed this and clarified that sentence.  Thanks for picking up the mistake.